eZ/Publish/Core/Repository/Repository.php

```diff
@@ -879,6 +744,28 @@ public function getFieldTypeService()
     }
 
     /**
+     * Get PermissionService.
+     *
+     * @return \eZ\Publish\API\Repository\PermissionService
+     */
+    public function getPermissionService()
+    {
+        if ($this->permissionService === null) {
+            $this->permissionService = new PermissionService(
+                $this,
+                $this->getUserService(),
```

Comment:
You are referring to for `getCurrentUser` right?
I'm somewhat leaning toward omitting that and only having `getCurrentUserRef` as this service does not really care about the full user.

This implies we should cover this use case somewhere higher up before we remove the deprecated `Repository->getCurrentuser()`.